webapp/pages/terms-and-conditions.vue

```diff
@@ -14,10 +15,16 @@
 
 <script>
 export default {
+  layout: 'blank',
   head() {
     return {
       title: this.$t('site.termsAc'),
     }
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-19T13:38:33Z" title="Friday, July 19th 2019, 3:38:33 pm +02:00">Jul 19, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,14 +1,72 @@
 <template>
   <div>
     <ds-space>
-      <ds-heading tag="h2">{{ $t('site.termsAc') }}</ds-heading>
+      <ds-heading tag="h2">{{ $t('site.termsAc') }} ( vom: {{ latestDate_TaC }})</ds-heading>
     </ds-space>
+    <div v-if="confirmed === true">
+      <ds-section class="hasConfirm" primary>
+        <ds-flex>
+          <ds-flex-item>
+            <ds-logo inverse></ds-logo>
+          </ds-flex-item>
+          <ds-flex-item width="4">
+            <ds-heading>you have confirmed</ds-heading>
+            <ds-text>
+              You have confirmed the current Terms and Conditions on
```


```suggestion
  {{ $t('terms-and-conditions.confirmed') }}
```
You want the text to be translated. You can also pass variables to the call of `$t`. Check out our examples by looking into our code, from folder `webapp` run `git grep '$t(`'` this should find all the lines of code where we use translations, including where we parameterize the translations.